src/maintenance/azext_maintenance/maintenance/maintenance_management_client.py

```diff
@@ -0,0 +1,106 @@
+# pylint: disable=too-many-instance-attributes,too-few-public-methods
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
```

Comment:
Usually we use 'vendored_sdk' as folder name instead of 'maintenance', you can reference to other extensions' folder structure.

Reply:
Is this a requirement? Why are the style tests not flagging this?